pandas/tests/dtypes/test_inference.py

```diff
@@ -495,6 +495,13 @@ class TestTypeInference(object):
     class Dummy():
         pass
 
+    def test_inferred_dtype_fixture(self, any_skipna_inferred_dtype):
+        # see pandas/conftest.py
```

Comment:
@jreback 
This is coming out of your review above:
> > I'm asking if you want me to move this particular fixture to `pandas/conftest.py` and then test it within the dtype tests (because this is effectively a dtype thing).

> Yes

And I don't get how adding this fixture is tied to code removal? I'm testing the `.str`-accessor on all the inferred dtypes to make sure it raises correctly, that's what I mainly need this fixture for.

That I'm testing the validity of the fixture in `test_inference.py` is for consistency, because it belongs there thematically (but could otherwise test that directly in the fixture constructor).

Reply:
@jreback 
This fixture would be a perfect candidate for splitting off a PR, but then, I'm afraid you're gonna say it doesn't do anything interesting (yet).

Do you want me to:
* split it up, and have the fixture being unusued until this PR is merged,
* or do want me to keep things logically together (i.e. in this PR)?